src/lib/components/composites/paper-components/PaperListEntry.svelte

```diff
@@ -28,8 +28,7 @@
         }
     };
```

Comment:
The `navigateToPaperView` function executes the same check as the `const href` part.
Can't you simply reuse the `href` variable?

```suggestion
    const navigateToPaperView = () => goto(href);
```

Reply:
Good point. Thank you.